package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "courseography",
+  "version": "0.2.0.0",
+  "repository": "git@github.com:Courseography/courseography.git",
+  "author": "David Liu <david@cs.toronto.edu>",
+  "license": "GPLv3",
+  "dependencies": {
+    "path": "^0.12.7",
```

Comment:
I feel like it might make more sense for these to be in `devDependencies`, while things like React and ReactDOM would be in here.